doc_source/intrinsic-function-reference-cidr.md

```diff
@@ -1,95 +1,140 @@
 # `Fn::Cidr`<a name="intrinsic-function-reference-cidr"></a>
 
-The intrinsic function `Fn::Cidr` returns the specified Cidr address block\.
+The intrinsic function `Fn::Cidr` returns an array of CIDR address blocks\. The number of CIDR blocks returned is dependent on the count parameter\.
 
-## Declaration<a name="w3ab2c21c28c16b5"></a>
+## Declaration<a name="w3ab2c21c28c37b5"></a>
 
-### JSON<a name="intrinsic-function-reference-cidr-syntax.json"></a>
+### JSON<a name="intrinsic-function-reference-getcidr-syntax.json"></a>
 
 ```
-{ "Fn::Cidr" : [ipBlock, count, sizeMask]}
+{ "Fn::Cidr" : [ipBlock, count, cidrBits] }
 ```
 
 ### YAML<a name="intrinsic-function-reference-cidr-syntax.yaml"></a>
 
 Syntax for the full function name:
 
 ```
-Fn::Cidr: [ ipBlock, count, sizeMask ] 
+Fn::Cidr:
+  - ipBlock
+  - count
+  - cidrBits
 ```
 
 Syntax for the short form:
 
 ```
-!Cidr [ ipBlock, count, sizeMask ]
+!Cidr [ ipBlock, count, cidrBits ]
 ```
 
-## Parameters<a name="w3ab2c21c28c16b7"></a>
+## Parameters<a name="w3ab2c21c28c37b7"></a>
 
 ipBlock  
-The user\-specified default Cidr address block\.
+The user\-specified CIDR address block to be split into smaller CIDR blocks\.
 
 count  
-The number of subnets' Cidr block wanted\. Count can be 1 to 256\.
+The number of CIDRs to generate. Valid range is between 1 and 256\.
 
-sizeMask  
-Optional\. The digit covered in the subnet\.
+cidrBits  
+The number of subnet bits for the CIDR\. For example, cidrBits of "8" will create a CIDR with a mask of "/24".
 
-## Return Value<a name="w3ab2c21c28c16b9"></a>
+**Note**
+Subnet bits is the inverse of subnet mask\. To workout the required host bits for a given subnet bits, subtract the subnet bits from 32 for IPv4 or 128 for IPv6\.
 
-The specified Cidr block\.
+## Return Value<a name="w3ab2c21c28c37b9"></a>
 
-## Example<a name="intrinsic-function-reference-cidr-examples"></a>
+An array of CIDR address blocks\.
 
-### <a name="intrinsic-function-reference-cidr-example1"></a>
+## Supported Functions<a name="w3ab2c21c28c37c13"></a>
 
-#### YAML<a name="intrinsic-function-reference-cidr-example1.yaml"></a>
+You can use the following functions in a `Fn::Cidr` function:
+
++ `[`Fn::Select`](intrinsic-function-reference-select.md)` 
+
```

Comment:
These aren't rendering as links correctly e.g:
```
`[`Fn::Select`](intrinsic-function-reference-select.md)` 
```
 renders to `[`Fn::Select`](intrinsic-function-reference-select.md)` 

If you remove the wrapping '`' it should sort it:
```
[`Fn::Select`](intrinsic-function-reference-select.md)
```
renders to
[`Fn::Select`](intrinsic-function-reference-select.md)

Reply:
I'm not sure why this rendering is showing up incorrectly. The latest commit that we're trying to merge is correctly rendering and using the style you suggested in this comment. See:

https://raw.githubusercontent.com/NightKhaos/aws-cloudformation-user-guide/pr-33/doc_source/intrinsic-function-reference-cidr.md